.github/workflows/check.yaml

```diff
@@ -23,45 +28,49 @@ jobs:
         python-version: ["3.8", "3.10"]
     with:
       python-version: ${{ matrix.python-version }}
-      working-directory: ./src
```

Comment:
Ah this is why it's failing.  It would be good if we could move the tox.ini file to the root of the repository here.  We may need to use `changedir` in the tox.ini - similar to what I'm trying to do at https://github.com/canonical/charm-advanced-routing/pull/44